Cargo.toml

```diff
@@ -11,6 +11,14 @@ exclude = [
     ".github/",
 ]
 
+[[bin]]
+name = "calendar-example"
+path = "src/calendar-example.rs"
```

Comment:
In the current form, this breaks the default behaviour of `cargo run` (it should run `wohnzimmer`). Since this is just an example, better leverage the cargo examples features instead: https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples

- remove the `[[bin]]` sections
- move `src/calendar-example.rs` to `examples/google-calendar.rs`
- move `dotenv` from `dependencies` to `dev-dependencies` (it's only used by the example and `dev-dependencies` get linked into examples)

Afterwards, you can run the example via `cargo run --example google-calendar`. And `cargo run` without arguments will start `wohnzimmer` as before.